src/Common/src/System/Collections/Generic/ReferenceEqualityComparer.cs

```diff
@@ -2,19 +2,27 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.Generic;
 using System.Runtime.CompilerServices;
 
-namespace System.Dynamic.Utils
+namespace System.Collections.Generic
 {
-    internal sealed class ReferenceEqualityComparer<T> : IEqualityComparer<T> where T : class
+    internal sealed class ReferenceEqualityComparer<T> : IEqualityComparer<T>
+        where T : class
     {
         internal static readonly ReferenceEqualityComparer<T> Instance = new ReferenceEqualityComparer<T>();
 
-        private ReferenceEqualityComparer() { }
+        private ReferenceEqualityComparer()
+        {
+        }
 
-        public bool Equals(T x, T y) => ReferenceEquals(x, y);
+        public bool Equals(T x, T y)
+        {
+            return ReferenceEquals(x, y);
+        }
 
-        public int GetHashCode(T obj) => RuntimeHelpers.GetHashCode(obj);
+        public int GetHashCode(T obj)
+        {
+            return RuntimeHelpers.GetHashCode(obj);
+        }
```

Comment:
Not a big deal, but why did you change these from being expression-bodied members?